### ISSUE TITLE
Bypass?

### ISSUE THREAD

**Issue by Nom-Youtube** [DarwinYouTube/Pirate-Rust-Cheat#1]
Im assuming we need a bypass, as by default we cant access the PID and create a handle due to EAC.

**Comment by DarwinYouTube** [comment]
> Im assuming we need a bypass, as by default we cant access the PID and create a handle due to EAC.

You're right. I recommend that you learn how to work with drivers and you can create a driver and enter a cheat inside the driver and it will work with the Rust game

**Comment by Nom-Youtube** [comment]
I noticed you made a bypass for the cracked rust, is it a similar concept?

**Comment by DarwinYouTube** [comment]
> I noticed you made a bypass for the cracked rust, is it a similar concept?

Nope. This is Anti-cheat the Simple on C#
He don't work with memory

**Comment by Nom-Youtube** [comment]
So I make a driver and add an entry point to the RustClient.exe?

**Comment by Nom-Youtube** [comment]
would work if I made a handle to lsass.exe (sub program has open handles to every active process) and use it to make a handle to RustClient.exe and inject via memory?

**Comment by DarwinYouTube** [comment]
> would work if I made a handle to lsass.exe (sub program has open handles to every active process) and use it to make a handle to RustClient.exe and inject via memory?

Try it man, there are many ways

**Comment by Nom-Youtube** [comment]
thanks ill let you know if I get something working!
